content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -6,6 +6,163 @@ card:
   name: reference
   weight: 60
 ---
+<style>
+  .tblFilter tr td {
```

Comment:
Thanks for your contribution!  Just a quick suggestion: to maintain clean and organised markdown files, it's best to avoid adding HTML and script code directly.